sdks/python/apache_beam/runners/common.py

```diff
@@ -570,6 +594,7 @@ def process_outputs(self, windowed_input_element, results):
         self.main_receivers.receive(windowed_value)
       else:
         self.tagged_receivers[tag].receive(windowed_value)
+    self.per_element_output_counter.update(output_element_output)
```

Comment:
Anything we do per-element is very performance sensitive. Profiles?

Reply:
I ran wordcount pipeline with 1gb input 10 times both with and without per_element_output_counter. I got result as following:
+------------|  with     | without |
avg(runtime) | 758.5s |  681s .   |
+------------------------------+